modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-ibm-storage_{context}"]
+= Setting up {ibm-cloud-title} Object Storage
+
+You can set up {ibm-cloud-title} Object Storage by using the {oc-first}.
+
+.Prerequisites
+
+* You have installed the latest version of `oc`. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
+
+* You have installed the latest version of the {ibm-cloud-title} Command Line Interface (AWS CLI). For more information, see "Getting started with the {ibm-cloud-title} CLI" in _{ibm-cloud-title} Docs_.
+
+* You are ready on {ibm-cloud-title}. For more information, see "Choosing a plan and creating an instance" in _{ibm-cloud-title} Docs_.
+
+** You have an {ibm-cloud-title} Platform account.
+** You have ordered an {ibm-cloud-title} Object Storage plan.
+** You have created an instance of {ibm-cloud-title} Object Storage.
+
+.Procedure
+
+. On {ibm-cloud-title}, create an object store bucket.
+
+. On {ibm-cloud-title}, create a service key with the following parameters for connecting to the object store bucket:
++
+[source,terminal]
+----
+$ ibmcloud resource service-key-create <tempo_bucket> Writer --instance-name <tempo_bucket> --parameters '{"HMAC":true}'
```

Comment:
Consider splitting this across lines so it can be seen w/o a scrollbar

Reply:
Thank your for the suggestion. Done.